tagstudio/src/qt/modals/drop_import.py

```diff
@@ -167,10 +167,11 @@ def duplicates_choice(self) -> int:
         dupes_to_show = self.duplicate_files
         if len(self.duplicate_files) > display_limit:
             dupes_to_show = dupes_to_show[0:display_limit]
+        
+        dupes_str = '\n    '.join(map(lambda path: str(path),dupes_to_show))
+        dupes_more = "\nand {len(self.duplicate_files)-display_limit} more " if len(self.duplicate_files) > display_limit else '\n'
```

Comment:
`dupes_more`  still has f-string syntax without the f-string

Reply:
Nice catch, resolved. Looks like I forgot to test it with >5 duplicate files